plugin/storage/grpc/README.md

```diff
@@ -0,0 +1,73 @@
+gRPC Storage Plugins
+====================
+gRPC Storage Plugins currently use the Hashicorp go-plugin. This requires the implementer of a plugin to develop the 
+"server" side of the go-plugin system.
+
+Implementing a plugin
+----------------------
```

Comment:
Actually this whole section is if someone wants to implement the plugin using the existing components in Jaeger like grpc server, storage API, proto-generated types, etc. I would make it clear, since it's not the only way and the more general requirement is "implement grpc server using `storage_v1.proto`"